src/_posts/languages/ruby/2000-01-01-start.md

```diff
@@ -155,18 +155,25 @@ end
 With these settings, you can change the number of web workers and threads by defining
 the `WEB_CONCURRENCY` and `MAX_THREADS` environment variables and restarting your app.
 
-### WEB_CONCURRENCY
-
-The recommended level of concurrency for the web server is defined automatically according
-to the available memory of the containers of your application. If you want to override
-this value, you can define the environment variable: `WEB_CONCURRENCY`.
-If you do not define this variable, the buildpack will set its value according to this table:
-
-| Container Memory (MB) | Default Concurrency |
-|-----------------------|---------------------|
-| 256                   | 1                   |
-| 512                   | 1                   |
-| 1024                  | 2                   |
-| 2048                  | 2                   |
-| 4096                  | 4                   |
-| 8192                  | 4                   |
+
+## WEB_CONCURRENCY
+
+The default level of concurrency is automatically defined, depending on the
+amount of memory available in the container running the app:
+
+| Container Size | Memory (MB) | Default Concurrency |
+| -------------: | ----------: |-------------------: |
+| S              | 256         | 1                   |
+| M              | 512         | 1                   |
+| L              | 1024        | 2                   |
+| XL             | 2048        | 2                   |
+| 2XL            | 4096        | 4                   |
```

Comment:
The previous table contained the `8192` memory. Shouldn't we keep it?

Reply:
Good question, our official public catalog includes our containers up to 4096 (https://scalingo.com/pricing). That's why I chose to remove the 3XL line from the table.
Let me know if you'd rather want to include it :)